runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -128,7 +121,7 @@ public void optimisticCommitUnsafe(long version) {
         this.version = version;
         // TODO: fix the stream view pointer seek, for now
         // read will read the tx commit entry.
-        sv.next();
+        //sv.seek(version);
```

Comment:
no, post commit doesn't work because we could have committed but there could be non-conflicting updates we would skip over due to how conflicts work